superset/assets/src/explore/AdhocFilter.js

```diff
@@ -45,7 +45,8 @@ export default class AdhocFilter {
       this.clause = adhocFilter.clause;
       this.sqlExpression = null;
     } else if (this.expressionType === EXPRESSION_TYPES.SQL) {
-      this.sqlExpression = adhocFilter.sqlExpression ||
```

Comment:
I'm not sure in what cases this would have been undefined if expressionType === sql. Gabe do you have more context here? Is checking for undefined sufficient or do I also need to check for nulls?